src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -382,10 +383,15 @@ internal void ApplyReplacementText(string replacementText, bool propagateEditImm
             VerifyNotDismissed();
             this.ReplacementText = replacementText;
 
+            var token = _asyncListener.BeginAsyncOperation(nameof(ApplyReplacementText));
```

Comment:
Nit: consider calling it asyncToken just since there's a lot of different token types bouncing around here.